qemu/tests/hotplug_mem.py

```diff
@@ -79,8 +80,13 @@ def start_test(self):
         if not callable(func):
             self.test.error("Unsupported memory operation '%s'" % operation)
         vm = self.env.get_vm(self.params["main_vm"])
-        vm.wait_for_login(timeout=login_timeout)
+        session = vm.wait_for_login(timeout=login_timeout)
         bootup_time = time.time() - vm.start_time
+        if self.params.get('os_type') == 'linux':
+            arg = "movable_node"
+            re_arg, _ = utils_test.check_kernel_cmdline(session, args=arg)
+            if re_arg:
+                utils_test.update_boot_option(vm, args_added=re_arg)
```

Comment:
`check_kernel_cmdline` here is  redundant.  In  `update_boot_option`,  it will `check_kernel_cmdline` first, then do next step.

Reply:
@yanan-fu Yes, thanks, updated.